requirements/main.in

```diff
@@ -5,15 +5,21 @@
 # After editing, update requirements/main.txt by running:
 #     make update-deps
 
+aiohttp
+aiodns
+chardet
```

Comment:
Yeah, this is irritating because those dependencies are really part of the aiohttp dependencies, which is why they're grouped. I'll add a comment to make that clearer.